gpt4all-chat/localdocsmodel.cpp

```diff
@@ -0,0 +1,43 @@
+#include "localdocsmodel.h"
+
+LocalDocsModel::LocalDocsModel(QObject *parent)
+    : QAbstractListModel(parent)
+{
+}
+
+int LocalDocsModel::rowCount(const QModelIndex &parent) const
+{
+    Q_UNUSED(parent);
```

Comment:
I don't know if there's an advantage to `Q_UNUSED`, if so please let me know. But would it be portable to just not define the argument name in the function params?

Reply:
Yes, it allows you to see what names are given which helps with readability of code.